docs/source/cli/examples.rst

```diff
@@ -227,19 +227,61 @@ The minimal command to create a simple order looks something like::
 If no toolchain or delivery details are specified, a basic order with download
 delivery will be placed for the requested bundle including the item id(s) specified.
 
-Additionally, optional toolchain & delivery details can be provided on the
-command line, e.g.:::
+In the place of `--id`, you can insert a Data search string. This will populate
+the list of IDs from the search. For example::
+
+    planet orders create --name "my order" \
+      --ids_from_search "--item-type PSScene3Band --date acquired gt 2017-02-14 --date acquired lt 2017-03-14" \
```

Comment:
It seems likely that someone creating an order for a date range might also want to apply a geometry filter. An example of `--ids_from_search` with both spatial (geometry filter) and temporal (date rnage) filtering would be helpful!

Reply:
makes sense, I'll update

Reply:
added spatial + temporal filter example to PR, and a couple new ways to reference geo things to my vocabulary :)